docs/source/index.rst

```diff
@@ -71,6 +79,21 @@ If you are using ferret for your work, please consider citing us!
         publisher = "Association for Computational Linguistics",
     }
 
+Also, ferret's Speech XAI functionalities are based on
```

Comment:
@g8a9 could you check this citation is in the correct format? (Newbie here!)

Reply:
yes indeed!